### PR TITLE
Make RichChoiceView use OverlayDropdown

### PR DESCRIPTION
**Pull request by sarthak-marwaha**

### CODE REVIEW

File: src/foam/u2/md/OverlayDropdown.js [hunk 1]
```diff
@@ -98,7 +103,24 @@ foam.CLASS({
...
+    {
+      class: 'Boolean',
+      name: 'styled',
+      value: true
+    },
+    {
+      class: 'Int',
+      name: 'parentEdgePadding',
+      documentation: `When set, the dropdown will stick itself to the parentEl's edge`,
+      value: -1
```
Comment (AlexeyGreer) [inline src/foam/u2/md/OverlayDropdown.js:116]: https://github.com/kgrgreer/foam3/blob/development/doc/guides/StyleGuide.md

Max line length of 80, your line is 81.

Comment (jlhughes) [reply src/foam/u2/md/OverlayDropdown.js:116]: Are you guys serious?!
You'd think you were brothers.

**Review by Nauna — APPROVED** [review]

File: src/foam/u2/md/OverlayDropdown.js (hunk 1 above)
Comment (kgrgreer) [reply src/foam/u2/md/OverlayDropdown.js:116]: What the style guide says:

"Line Length
Line lengths should be 80 characters or less, except for embedded data, like templates or sprites, or when modifying the code to make it fit in less than 80 characters would actually makes it less readable."

I would argue that going over by one character makes things more readable than splitting onto two lines.

Comment (sarthak-marwaha) [reply src/foam/u2/md/OverlayDropdown.js:116]: I agree

**Review by AlexeyGreer — APPROVED** [review]

**Pull request merged**